lib/ui/other/route_utils.dart

```diff
@@ -153,7 +154,14 @@ class RouteUtils {
           onPageClose: () => Get.back(),
         ),
         transition: transition,
+        preventDuplicates: true,
       );
 
   static bool isLoggedIntoZuvio() => ZAuthController.to.isLoggedIntoZuvio();
+
+  static Future<void> launchCameraPage(String courseId) => Get.to(
```

Comment:
This `Future` seems unnecessary.

```suggestion
  static void launchCameraPage(String courseId) => Get.to(
```